File: utentes/static/models/Estado.js
Backbone.SIXHIARA = Backbone.SIXHIARA || {};
Backbone.SIXHIARA.Estado = Backbone.UILib.Domain.extend({
    urlRoot: "/api/domains/licencia_estado",

    defaults: {
        category: "licencia_estado",
        alias: null,
        text: null,
        order: 0,
        parent: "precampo",
    },
});

File: utentes/models/estado.py
from sqlalchemy import Column, Integer, Text
from sqlalchemy.dialects.postgresql import ARRAY

from .base import PGSQL_SCHEMA_DOMAINS, DeclarativeBase


class Estado(DeclarativeBase):
    __tablename__ = "licencia_estado"
    __table_args__ = {"schema": PGSQL_SCHEMA_DOMAINS}

    category = Column(Text, nullable=False, primary_key=True)
    key = Column(Text, nullable=False, primary_key=True)
    value = Column(Text)
    ordering = Column(Integer)
    parent = Column(Text, primary_key=True)
    tooltip = Column(Text)
    app = Column(ARRAY(Text), doc="app")

    def __json__(self, request):
        return {
            "category": self.category,
            "text": self.key,
            "alias": self.value,
            "order": self.ordering,
            "parent": self.parent,
            "tooltip": self.tooltip,
        }
